src/plugins/intel_cpu/src/node.cpp

```diff
@@ -1092,29 +1092,9 @@ void Node::setDynamicBatchLim(int lim) {
 
 void Node::appendPostOpArgs(const dnnl::primitive_attr& attr,
                                   std::unordered_map<int, dnnl::memory>& primArgs,
-                                  const std::vector<MemoryPtr>& postOpsArgs) {
-    constexpr size_t maxPrimArgsCapacity = 32;
-    auto post_ops = attr.get_post_ops();
-    int idx = 0;
-    for (int i = 0; i < post_ops.len(); i++) {
-        if (one_of(post_ops.kind(i), dnnl::primitive::kind::binary, dnnl::primitive::kind::depthwise, dnnl::primitive::kind::quantization)) {
-            if (idx >= postOpsArgs.size()) {
-                IE_THROW() << "Cannot initialize primitive arguments: invalid post-ops data pointers count";
-            }
-            // oneDNN has implicit limitation on number of supported post ops arguments
-            if (i >= maxPrimArgsCapacity) {
-                IE_THROW() << "Cannot initialize primitive arguments: post-ops data pointers count exceed max capacity";
-            }
-
-            primArgs[DNNL_ARG_ATTR_MULTIPLE_POST_OP(i) | DNNL_ARG_SRC_1] = postOpsArgs[idx++]->GetPrimitive();
-        } else if (post_ops.kind(i) == dnnl::primitive::kind::convolution) {
-            if (idx + 1 >= postOpsArgs.size()) {
-                IE_THROW() << "Cannot initialize primitive arguments: invalid post-ops data pointers count";
-            }
-
-            primArgs[DNNL_ARG_ATTR_POST_OP_DW | DNNL_ARG_WEIGHTS] = postOpsArgs[idx++]->GetPrimitive();
-            primArgs[DNNL_ARG_ATTR_POST_OP_DW | DNNL_ARG_BIAS] = postOpsArgs[idx++]->GetPrimitive();
-        }
+                                  const std::unordered_map<int, MemoryPtr>& postOpsArgs) {
+    for (auto & entry : postOpsArgs) {
```

Comment:
Looks like we were handling it by simply throw exceptions, then why not leave the check to oneDNN since it knows it's own implementation better than anybody else?